src/components/authentication/CreateProfilePage.tsx

```diff
@@ -166,11 +185,19 @@ export default function CreateAccountPage({
           onSubmitEditing={() =>
             lastNameTextInput.current && lastNameTextInput.current.focus()
           }
+          onChangeText={(text) => {
+            setValue("firstName", text);
+          }}
+          errorMessage={errors.firstName ? formErrors.firstName : ""}
         />
         <ThemedInput
           placeholder="Smith"
           label="Last Name"
           ref={lastNameTextInput}
+          onChangeText={(text) => {
+            setValue("lastName", text);
+          }}
+          errorMessage={errors.lastName ? formErrors.lastName : ""}
```

Comment:
can you double check this one? I think the logic is backwards here

Reply:
This logic is correct, errors.lastName tells you what type of error occurred for lastName. In this case, we only have one type of error which is for minLength. formErrors.lastName will yield the human readable message for the error. Thus the ternary operator means "if there in an error for last name, return the human readable error message, otherwise return nothing"